app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -109,6 +118,27 @@ def parse_build_pod(build_pod)
                                                            build_pod.status.config.try(:name))
         new_result
       end
+
+      def parse_template_parameters(parameters)
```

Comment:
We still use symbols here. Note that the strings are in the hash that being parsed (this is how we get it from Openshift).